components/intl.rst

```diff
@@ -240,6 +240,10 @@ which defaults to the current default locale::
     $languages = Languages::getNames('de');
     // => ['ab' => 'Abchasisch', ...]
 
+You can also check if a given language code is valid::
+
+    $isValidLanguage = Languages::exists($languageCode);
```

Comment:
$languageCode is undefined, maybe use a real string here?

Reply:
If I use a real string ... then I need to add a comment that the string argument is the "language code". Also, if we add a real string, we may need to show two cases: valid and invalid strings.

Do you really think that someone reading that code can't understand that `$languageCode` is a variable where you store the language code to check if it's valid?

Reply:
Fair enough, your explanation makes sense to me, so let’s keep it like this 🚀